### PR TITLE
New Serverless Pattern - SQS to SQS message copy with filter using EventBridge Pipe using SAM

### PR DESCRIPTION
**Pull request by biswanathmukherjee**
*Issue #, if available:*

*Description of changes:*
The patterns shows the filter capability of the Amazon EventBridge Pipe while copying data from one Amazon SQS queue to another queue using SAM.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: eventbridge-pipes-sqs-to-sqs-with-data-filter/README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+* We will validate received messages on `target-queue` using `receive-messages-from-target-queue.sh`. Only the messages with `red` and `blue` values for `color` attribute will be available in `target-queue`.
+
+
+Please refer to the architecture diagram below:
+
+![End to End Architecture](image/architecture.png)
+
+## Testing
+
+1. Run the script `receive-messages-from-target-queue.sh` to send four messages to the `source-queue` with a payload in `{"id":1, "color":"<value>"}` format having `red`, `blue`, `green` and `yellow` as color values. The script will ask for the queue URL and the region. Please provide the `SourceQueueURL` value received from the outout of the deployment step and your deployment region.
```
Comment (parikhudit) [inline eventbridge-pipes-sqs-to-sqs-with-data-filter/README.md:50]: Typo: `receive-messages-from-target-queue.sh`

**Comment by parikhudit** [comment]
@biswanathmukherjee could you rename README and the folder with something like `eventbridge-pipes-sqs-to-sqs-with-data-filter`?

**Comment by biswanathmukherjee** [comment]
> @biswanathmukherjee could you rename README and the folder with something like `sqs-eventbridgepipes-sqs-msg-copy-data-filter-sam`?

I have changed the folder name to sqs-eventbridgepipes-sqs-msg-copy-data-filter-sam.

File: sqs-eventbridgepipes-sqs-msg-copy-data-filter-sam/README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+   ```
+   git clone https://github.com/aws-samples/serverless-patterns
+   ```
+2. Change directory to the pattern directory:
+   ```
+   cd eventbridge-pipes-sqs-to-sqs-with-data-filter
```
Comment (parikhudit) [inline sqs-eventbridgepipes-sqs-msg-copy-data-filter-sam/README.md:24]: old folder name.. please update

Comment (parikhudit) [reply sqs-eventbridgepipes-sqs-msg-copy-data-filter-sam/README.md]: nevermind, I fixed it.

**Pull request merged**